packages/palette/src/elements/Checkbox/Checkbox.tsx

```diff
@@ -49,7 +49,8 @@ export const Checkbox: React.FC<CheckboxProps> = ({
   }
 
   const handleKeyPress = (event: React.KeyboardEvent<HTMLDivElement>) => {
-    if (event.code === "Space" && isSelectable) {
+    if (event.key === " " && isSelectable) {
```

Comment:
Weird thing: in Force event.code is undefined; event.key is fine in this case/more widely supported anyway; but odd.